src/constants/constants.js

```diff
@@ -16,6 +16,13 @@ export const CONSTANTS = {
         VARROCK: "VARROCK",
         BARBARIAN_VILLAGE: "BARBARIAN_VILLAGE",
     },
+    PREREQUISITES: {
```

Comment:
I think this is more confusing than inputting the scene constant itself. Before I looked at constants I thought you duplicated them for some reason

Reply:
I like it because for all the places prereqs may be used we only have to think about them once when making this. Helps with code readability as well - a good example in this PR is [here](https://github.com/EthanVieira/rs-clicker/blob/a1d68f266fa8b01f3b8e9daaf3f61418363b7782/src/cookie-io.js#L182). If i were to try and replace that with scene constants it would be more code and more confusing. Also, if there is ever a level with multiple prereqs then it will be very helpful to have them combined in one place.